src/features/Students/StudentsPage/_test_/reducer.test.jsx

```diff
@@ -5,7 +5,7 @@ import {
   UPDATE_CURRENT_PAGE,
 } from 'features/Students/actionTypes';
 import { RequestStatus } from 'features/constants';
-import reducer from 'features/Instructors/InstructorsPage/reducer';
+import reducer from 'features/Students/StudentsPage/reducer';
```

Comment:
Why this change? @AuraAlba

Reply:
It was using the incorrect reducer, was the same definition and didn't give error so for the moment I import the correct reducer but that will be removed in the other ticket